config/routes.rb

```diff
@@ -13,6 +13,25 @@
     get '/users/sign_out' => 'users/sessions#destroy'
   end
 
+
+  resources :orders do
+    collection do
+      get 'all_orders'
+    end
+    member do
+      get 'single_order'
+    end
+    member do
+      put 'mark_complete_order'
+    end
+    member do
+      put 'cancel_order'
+    end
+    member do
+      put 'mark_paid_order'
+    end
```

Comment:
Why?
you can list these routes in single block

Reply:
Done